Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -335,11 +340,17 @@ bool MainWindow::Stop()
                                            "may be lost if you stop the current emulation "
                                            "before it completes. Force stop?") :
                                         tr("Do you want to stop the current emulation?"));
+
+    m_inquiry_in_progress = false;
+
     if (pause)
       Core::SetState(state);
 
     if (confirm != QMessageBox::Yes)
+    {
+      m_exit_requested = false;
```

Comment:
I like! This is much easier to think about. 👍 

This `m_exit_requested` line should be removed now.